x-pack/plugins/observability_solution/observability/common/utils/alerting/get_related_alerts_query.ts

```diff
@@ -5,28 +5,102 @@
  * 2.0.
  */
 
+import { ALERT_RULE_UUID } from '@kbn/rule-data-utils';
 import type { Group } from '../../typings';
 
 export interface Query {
   query: string;
   language: string;
 }
+export interface Field {
+  name: string;
+  value: string;
+}
+interface Props {
+  tags?: string[];
+  groups?: Group[];
+  ruleId?: string;
+  sharedFields?: Field[];
+}
+
+// APM rules
+export const SERVICE_NAME = 'service.name';
+// Synthetics rules
+export const MONITOR_ID = 'monitor.id';
+// - location
+export const OBSERVER_NAME = 'observer.name';
+// Inventory rule
+export const HOST = 'host.name';
```

Comment:
would host.ip also make sense here?

Reply:
Maybe, I need to check a bit more to see if this field is useful for most of the scenarios or not. (Added a new tab to this [spreadsheet](https://docs.google.com/spreadsheets/d/1tIe0TwXgLaTJho4jQ9gaXNxiMPm0aPjoA_afsifpGjY/edit?gid=595309499#gid=595309499), "other fields" tab, feel free to add more fields that you think might be useful)

I started with a small set of fields, and we can extend it in the future.